src/envoy/mixer/utils.cc

```diff
@@ -54,8 +54,14 @@ bool GetSourceUser(const Network::Connection* connection, std::string* user) {
   if (connection) {
     Ssl::Connection* ssl = const_cast<Ssl::Connection*>(connection->ssl());
     if (ssl != nullptr) {
-      *user = ssl->uriSanPeerCertificate();
-      return true;
+      std::string result = ssl->uriSanPeerCertificate();
+      std::string prefix = "spiffe://";
+      // Strip out the prefix "spiffe://" in the identity.
+      std::size_t found = result.find(prefix);
```

Comment:
use compare like this

https://github.com/istio/mixerclient/blob/master/src/attributes_builder.cc#L34

find() is very expensive